test/runner/run.js

```diff
@@ -288,11 +281,7 @@ export async function run( {
 
 	try {
 		console.log( `Starting Run ${ runId }...` );
-		if ( browserstack ) {
-			await runAllBrowserStack( { verbose } );
-		} else {
-			await runAllSelenium( { concurrency, verbose } );
```

Comment:
Do we abandon the concurrency option?

Reply:
No, but this argument was being ignored even before this. Instead, `concurrency` is read from the test objects when running them.